lib/skc_codegen/src/gen_exprs.rs

```diff
@@ -262,7 +266,10 @@ impl<'hir, 'run, 'ictx> CodeGen<'hir, 'run, 'ictx> {
         let then_block_end = self.builder.get_insert_block().unwrap();
         // IfElse:
         self.builder.position_at_end(else_block);
+        let () = println!("{:?}", ctx.lvars);
```

Comment:
[FYI] For debugging, `dbg!` macro is handy. eg.`dbg!(&ctx.lvars);` will print

```
[lib/skc_codegen/src/gen_exprs.rs:256] &ctx.lvars = {}
```

Reply:
I didn't know for dbg!, thanks you